lib/Clownface.js

```diff
@@ -103,8 +103,8 @@ class Clownface {
     return this.values.join()
   }
 
-  node (values, { type, datatype } = {}) {
-    values = toTermArray(values, type, datatype) || [null]
+  node (values, { type, datatype, language } = {}) {
+    values = toTermArray(values, type, datatype || language) || [null]
```

Comment:
I don't see the `language` parameter used anywhere.

Reply:
It's passed to `toTermArray`, `term` and `literal` and in the end it's used [here](https://github.com/rdf-ext/clownface/blob/terms/lib/term.js#L22).